src/encode/rangecoder.rs

```diff
@@ -274,40 +319,43 @@ mod test {
 
     #[test]
     fn test_encode_decode_bittree_zeros() {
-        for num_bits in 0..16 {
-            encode_decode_bittree(num_bits, &[0; 10000]);
-        }
+        seq!(NUM_BITS in 0..16 {
```

Comment:
Since `NUM_BITS` is now compile-time, the `seq!` macro is used for these tests instead.